### PR TITLE
SME QE and Peer review of Bxmsdoc 3565 masterShowcase App for case management for 7.2

### PR DESCRIPTION
**Pull request by michelehaglund**
Original Jira: https://issues.jboss.org/browse/BXMSDOC-3565
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-3565/

### CODE REVIEW

File: doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc [hunk 1]
```diff
@@ -18,24 +12,17 @@ endif::[]
 * The IT Orders project is deployed to {KIE_SERVER}.
 * The Showcase application `.war` file has been deployed alongside {CENTRAL}.
 
-.Procedure 
-. With the IT Orders project deployed and running in the {KIE_SERVER}, in a web browser, navigate to the Showcase login page `http://localhost:8080/rhpam-case-mgmt-showcase/`. 
+.Procedure
+. With the *IT_Orders_New* project deployed and running in the {KIE_SERVER}, in a web browser, navigate to the Showcase login page `http://localhost:8080/rhpam-case-mgmt-showcase/`.
```
Comment (cristianonicolai) [inline doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc]: In this context, to quickly get started with the showcase, I believe we want to encourage to use the out-of-box example, which is _IT_Orders_.

Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc]: Changed back to the sample

File: doc-content/enterprise-only/cases/case-management-showcase-application-con.adoc [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 
-The Showcase application is included in the {PRODUCT} distribution to demonstrate the capabilities of case management in an application environment. Showcase is intended to be used as a proof of concept that aims to show the interaction between business process management (BPM) and case management. You can use the application to start/close, monitor, and interact with cases.
+Showcase must be installed in addition to the {CENTRAL} application and {KIE_SERVER}. The Showcase application is required to start new case instances, however the case work is still performed in {CENTRAL}.
 
-Showcase must be installed in addition to the {CENTRAL} application and {KIE_SERVER}. The Showcase application is required to start new case instances, however the case work is still performed in {CENTRAL}. 
-
-After a case instance is created and is being worked on, you can monitor the case in the Showcase by clicking the case in the *Case List* to open the case *Overview* page. 
+After a case instance is created and is being worked on, you can monitor the case in the Showcase by clicking the case in the *Case List* to open the case *Overview* page.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-showcase-application-con.adoc]: Did mean to say Showcase application here?

File: doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc [hunk 3]
```diff
@@ -15,43 +15,35 @@ image::cases/itorders-case-start.png[]
...
-xref:case-management-starting-itorders-in-showcase-proc-case-management-showcase[].
-endif::[]
+As the user `wbadmin`, you have started an *IT_Orders* case in the Showcase application.
 
 .Procedure
 
-. Log out of {CENTRAL}, and log back in as a user that belongs to the `supplier` group. This example has the user `Lenovo` configured as a member of the `supplier` group.
-+
-image::cases/showcase-lenovo.png[]
+. Log out of {CENTRAL}, and log back in as a user that belongs to the `supplier` group.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc]: Delete comma

File: doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc [hunk 4]
```diff
@@ -15,43 +15,35 @@ image::cases/itorders-case-start.png[]
...
+. Click *Complete*.
 . In Showcase, click *Refresh* in the upper-right corner. Notice that the `Prepare hardware task` user task and the `Hardware spec ready` milestone appear in the *Completed* column.
 +
 image::cases/case-management-ordered.png[]
 +
-. Log out of {CENTRAL} and log back in again as `Katy`. 
-. Go to *Menu* -> *Track* -> *Task Inbox*. Open the `Manager approval` task for Katy. 
-.. Click *Claim*, then *Start*. 
+. In {CENTRAL} go to *Menu* -> *Track* -> *Task Inbox*. Open the `Manager approval` task for *wbadmin*.
+.. Click *Claim*, then *Start*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc]: I think we have to say 
Click Claim and then click Start.

File: doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc [hunk 5]
```diff
@@ -15,43 +15,35 @@ image::cases/itorders-case-start.png[]
...
 +
 image::cases/case-management-manager-approval-process.png[]
 +
 .. Refresh the Showcase page to see that the `Manager approval` task and the `Manager decision` milestone are in the *Completed* column. The *Milestones* pane in the lower-left corner of the Showcase overview page also shows the completed and pending milestones.
 +
 image::cases/showcase-milestones-ordered.png[]
-. Log out of {CENTRAL} and log back in again as `Aimee`. 
-. Go to *Menu* -> *Manage* -> *Tasks*. Click the `Place order` task to open it. 
-.. Click *Claim*, then *Start*. 
+. In {CENTRAL} go to *Menu* -> *Manage* -> *Tasks*. Click the `Place order` task to open it.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc]: Add comma after Central

File: doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc [hunk 6]
```diff
@@ -15,43 +15,35 @@ image::cases/itorders-case-start.png[]
...
+.. Click *Claim*, then *Start*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc]: AB4

File: doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc [hunk 7]
```diff
@@ -60,33 +52,33 @@ The process instance diagram now shows the following case progress:
...
 .. In the *Edit* window, type `true` and click *Save*.
 +
-.. Refresh the Showcase page. Note that the `Milestone 3: Delivered to customer` milestone is shown as *Completed*. All milestones under the *Milestones* pane in the lower-left corner are shown as complete. 
+.. Refresh the Showcase page. Note that the `Milestone 3: Delivered to customer` milestone is shown as *Completed*. All milestones under the *Milestones* pane in the lower-left corner are shown as complete.
 +
 The final task of the IT Orders case, `Customer satisfaction survey` is shown under *In progress*.
 +
 image::cases/itorders-customer-survey.png[]
-. As Aimee, open {CENTRAL} and go to *Menu* -> *Track* -> *Task Inbox*. Click the `Customer satisfaction survey` task to open it. 
+. In {CENTRAL} and go to *Menu* -> *Track* -> *Task Inbox*. Click the `Customer satisfaction survey` task to open it.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc]: In central do what? Missing verb.

File: doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc [hunk 8]
```diff
@@ -1,52 +1,45 @@
...
-These roles are configured at the case definition level:
-
-.ITOrders Case Roles
-image::cases/case_roles.png[Case Roles]
-
-Assign users or groups to these roles when starting a new case file instance.
-
-.Prerequisites 
+.Procedure
+. Select *Order for IT hardware* case instance from the list to open it.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc]: Select the Order...?

File: doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc [hunk 9]
```diff
@@ -1,52 +1,45 @@
...
+. Under *Overview* -> *Actions* -> *Available*, click image:cases/dotdotdotbutton.png[] next to *New user task* or *New process task* to add a new task or process task.
++
+.Showcase dynamic actions
+image::cases/showcase-dynamic-actions.png[]
+. Click start to create a *New user task* and input the following values:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc]: Enter/input - I get it. You like input :-)

File: doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc [hunk 10]
```diff
@@ -1,52 +1,45 @@
...
-<<case-management-install-and-login-to-showcase-proc-case-management-getting-started>>.
-endif::[]
-ifeval::["{context}" == "case-management-design"]
-<<case-management-install-and-login-to-showcase-proc-case-management-design>>.
-endif::[]
-ifeval::["{context}" == "case-management-showcase"]
-<<case-management-install-and-login-to-showcase-proc-case-management-showcase>>.
-endif::[]
+. Click *Add Task*.
+. Click start to create a *New process task* and input the following values:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc]: *Start*?

File: assemblies/assembly_showcase-application/main.adoc [hunk 11]
```diff
@@ -20,7 +20,7 @@ Use this document to install the Showcase application and start a case instance
 
 .Prerequisites
 
-* {EAP_LONG} 7.1.0 is installed. For installation information, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html-single/installation_guide/[_Red Hat JBoss EAP 7.1.0 Installation Guide_].
+* {EAP_LONG} 7.2.0 is installed. For installation information, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html-single/installation_guide/[_Red Hat JBoss EAP 7.2.0 Installation Guide_].
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply assemblies/assembly_showcase-application/main.adoc]: Done :)

File: doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc [hunk 12]
```diff
@@ -1,65 +1,45 @@
 [id='case-management-dynamic-tasks-proc']
-= Starting dynamic tasks and processes
+= Starting the Order for IT Hardware case
 
-You can add dynamic tasks and processes to a case during run time. Dynamic actions are a way to address changing situations, where an unanticipated change during the case requires a new task or process to be incorporated into the case. 
+Start a new case instance for the *IT_Orders* sample case management project in the Showcase application.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc]: Done

File: doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc [hunk 13]
```diff
@@ -1,65 +1,45 @@
...
-+
-image::cases/showcase-dynamic-user-task.png[]
-* To create a dynamic process task, start a *New process task* and complete the required information:
-+
-image::cases/showcase-dynamic-process-task.png[]
-. To view a dynamic user task in {CENTRAL}, click *Menu* -> *Track* -> *Task Inbox*. The user task that was added dynamically using the Showcase application appears in the *Task Inbox* of users assigned to the task during task creation.
-+
-image::cases/task-inbox-dynamic-task.png[]
+.Procedure
+. With the *IT_Orders* project deployed and running in the {KIE_SERVER}, in a web browser, navigate to `http://localhost:8080/rhpam-case-mgmt-showcase/`.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc]: I assume you meant the comma. I removed it.

File: doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc [hunk 14]
```diff
@@ -1,65 +1,45 @@
...
 +
-.. Click the dynamic task in the *Task Inbox* to open the task. A number of action tabs are available from this page.
-.. Using the actions available under the task tabs, you can begin working on the task.
+Alternatively, if you have configured {CENTRAL} to display the *Apps launcher* button, use it to open a new browser window with the Showcase login page.
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc]: I concur. Changed as suggested.

File: doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc [hunk 15]
```diff
@@ -1,65 +1,45 @@
...
 +
-.. In the Showcase application, click the refresh button in the upper-right corner. Case tasks and processes that are in progress appear under *Overview* -> *Actions* -> *In progress*.
-.. When you have completed working on the task, click the *Complete* button under the *Work* tab.
+image::cases/apps-launcher-showcase-button.png[]
+
+. Log in to the Showcase application using your {CENTRAL} login credentials.
+. Click *Start Case*.
+. Select the *Order for IT hardware* case name.
+. Input `wbadmin` in the *Case Owner* field, and complete the following role assignments:
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-dynamic-tasks-proc.adoc]: Not sure about this. We are telling them to add info in to a field.

File: doc-content/enterprise-only/cases/case-management-install-and-login-to-showcase-proc.adoc [hunk 16]
```diff
@@ -14,57 +14,59 @@ The Showcase application is included with the {PRODUCT} {PRODUCT_VERSION} distri
...
 endif::DM[]
-file. The 
+file. The
 ifdef::PAM[]
-(`rhpam-7.0-case-mgmt-showcase-eap7-deployable.zip`).
+(`rhpam-7.2-case-mgmt-showcase-eap7-deployable.zip`).
 endif::PAM[]
 ifdef::DM[]
-(`rhdm-7.0-case-mgmt-showcase-eap7-deployable.zip`).
+(`rhdm-7.2-case-mgmt-showcase-eap7-deployable.zip`).
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-install-and-login-to-showcase-proc.adoc]: We should chat about this moving forward. I really struggled with some of Gemma's settings. Leaving as is for now.

File: doc-content/enterprise-only/cases/case-management-install-and-login-to-showcase-proc.adoc [hunk 17]
```diff
@@ -14,57 +14,59 @@ The Showcase application is included with the {PRODUCT} {PRODUCT_VERSION} distri
...
+`_TEMP_DIR/rhdm-7.2-case-mgmt-showcase-eap7-deployable/jboss-eap-7.2`
 endif::DM[]
  directory to `__EAP_HOME__`.
 +
 When asked to overwrite files or merge directories, select *Yes*.
 +
 WARNING: Ensure the names of the {PRODUCT} deployments you are copying do not conflict with your existing deployments in the {EAP} instance.
 
-. Add the following system property to your deployment’s EAP_HOME/jboss-eap-7.1/standalone/configuration/standalone-full.xml file:
+. Add the following system property to your deployment’s EAP_HOME/jboss-eap-7.2/standalone/configuration/standalone-full.xml file:
```
[the remark this answers is not in the document]
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-install-and-login-to-showcase-proc.adoc]: Cool. Thanks

File: doc-content/enterprise-only/cases/case-management-showcase-application-con.adoc (hunk 2 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-showcase-application-con.adoc]: Yes. Done.

File: doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc (hunk 3 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc]: Done

File: doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc (hunk 4 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc (hunk 5 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc]: commas in, commas out. Geez

File: doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc (hunk 6 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc]: got it.

File: doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc (hunk 7 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc]: fixed

File: doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc (hunk 8 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc]: the!

File: doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc (hunk 9 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc]: I'm so confused

File: doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc (hunk 10 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc]: Corrected.

**Pull request merged**